src/main/java/task/Event.java

```diff
@@ -0,0 +1,33 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Event extends Task {
+
+    protected LocalDateTime ldt;
+    public String at;
```

Comment:
Class variables should not be public.
![image](https://user-images.githubusercontent.com/80502407/151959418-4756526e-dfb5-49d8-abfb-1670c316fcb5.png)